addons/LPCAnimatedSprite/LPCAnimatedSprite2D.gd

```diff
@@ -99,7 +99,7 @@ func AddAnimation(spriteSheet:LPCSpriteSheet, spriteFrames:SpriteFrames, animati
 		spriteFrames.remove_animation(animationData.Name)
 		
 	spriteFrames.add_animation(animationData.Name)
-	var frameStart = animationData.FrameCount -1 if animationData.Reverse else animationData.Col
+	var frameStart = animationData.FrameCount -1 if animationData.Reverse else 0
```

Comment:
Hi! If the animationData.Col is wrong maybe it should be changed on the LPCSpriteSheet.gd, where it is instanced:

```gdscript
LPCAnimationData.new(8,"WALK_UP",8, 1,true),
LPCAnimationData.new(8,"WALK_LEFT",9, 1,true),
LPCAnimationData.new(8,"WALK_DOWN",10, 1,true),
LPCAnimationData.new(8,"WALK_RIGHT",11, 1,true),
```
to

```gdscript
LPCAnimationData.new(8,"WALK_UP",8, 0,true),
LPCAnimationData.new(8,"WALK_LEFT",9, 0,true),
LPCAnimationData.new(8,"WALK_DOWN",10, 0,true),
LPCAnimationData.new(8,"WALK_RIGHT",11, 0,true),
```
the number 1 here is for the offset

Reply:
You then end up getting the idle frame as the first frame.

Reply:
I see, as I cannot test it more deeply, I'll accept the MR, so please can you update the version number too? If it's working on the 4.2.2, the version can be 4.2.2.1